app/views/repositories/show.html.erb

```diff
@@ -8,7 +8,11 @@
 
 <section id="show-repository">
 
+
   <section id="repository-photos">
+    <% if @repository.youtube_link %>
+      <%= youtube_video(@repository.youtube_link) %>
```

Comment:
Is there a way to check if the provided link is in fact a youtube video? I'm thinking either a check that "youtube.com" is part of the provided link. but then if they provide a link for a channel it won't be valid as well. But I suggest taking a look into it.

Reply:
I'm using a gem that return nil if it's not a youtube video link.

Reply:
Oh okay I see. that's good.